finagle-redis/src/test/scala/com/twitter/finagle/redis/commands/sentinel/SentinelClientIntegrationSuite.scala

```diff
@@ -0,0 +1,225 @@
+package com.twitter.finagle.redis.integration
+
+import com.twitter.conversions.time._
+import com.twitter.finagle.redis.naggati.SentinelClientTest
+import com.twitter.finagle.redis.tags.{ RedisTest, ClientTest }
+import com.twitter.finagle.util.DefaultTimer
+import com.twitter.util.{ Await, Duration, Future, Time }
+import com.twitter.finagle.redis.util.{ CBToString, StringToChannelBuffer, Sentinel }
+import org.jboss.netty.buffer.ChannelBuffer
+import org.junit.Ignore
+import org.junit.runner.RunWith
+import org.scalatest.junit.JUnitRunner
+import java.io.File
+import java.net.InetAddress
+
+@Ignore
+@RunWith(classOf[JUnitRunner])
+final class SentinelClientIntegrationSuite extends SentinelClientTest {
+
+  implicit def s2cb(s: String) = StringToChannelBuffer(s)
+  implicit def cb2s(cb: ChannelBuffer) = CBToString(cb)
+
+  val sentinelCount = 3
+  val masterCount = 2
+  val slavesPerMaster = 2
+  val count = masterCount * (slavesPerMaster + 1)
+  val master0 = masterName(0)
+  val master1 = masterName(1)
+  val noSuchMaster = masterName(999)
+
+  override def beforeAll(): Unit = {
+    super.beforeAll()
+    for {
+      i <- 0 until masterCount
+      j <- 1 to slavesPerMaster
+    } withRedisClient(i + masterCount * j) { client =>
+      val (host, port) = hostAndPort(redisAddress(i))
+      Await.ready(client.slaveOf("127.0.0.1", port))
+    }
+  }
+
+  private def masterName(i: Int) = "master" + i
+
+  private def waitUntil(msg: String)(check: => Boolean) = {
```

Comment:
github shows that this line is gone, but I'm not sure how that could be possible . . . weird.